src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -453,6 +453,14 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
             isMarkdown={true}
           />
         )}
+        <GlobalMessage
+          style={styles.purchaseInformation}
+          globalMessageContext={GlobalMessageContextEnum.appPurchaseInformation}
+          textColor="secondary"
```

Comment:
Is `textColor="secondary"` correct? 

From Figma:
![Screenshot 2024-04-02 at 11 24 31](https://github.com/AtB-AS/mittatb-app/assets/675421/2e3885f6-7e3d-401d-9203-1de430284c95)

Reply:
Updated, was using non-subtle global message, so the color change wasn't noticeable. Thanks for pointing it out!